crates/diesel_models/src/payment_attempt.rs

```diff
@@ -81,11 +62,19 @@ pub struct PaymentAttempt {
     pub profile_id: id_type::ProfileId,
     pub organization_id: id_type::OrganizationId,
     pub card_network: Option<String>,
+    pub payment_method_type_v2: Option<storage_enums::PaymentMethod>,
```

Comment:
Change the Enums too for `PaymentMethod` and `PaymentMethodType` to `PaymentMethodTypeV2` and `PaymentMethodSubType`.